store/database.go

```diff
@@ -12,26 +12,26 @@ type Reader interface {
 	// Has retrieves if a key is present in the key-value data store.
 	//
 	// Note: <key> is safe to modify and read after calling Has.
-	Has(storeKey string, key []byte) (bool, error)
+	Has(storeKey []byte, key []byte) (bool, error)
```

Comment:
this applies to all APIs btw. SInce the types are the same, you can omit all but the last.
```suggestion
	Has(storeKey, key []byte) (bool, error)
```